tests/transports/epics/pva/test_pva_util.py

```
@@ -0,0 +1,26 @@
+import pytest
+
+from fastcs.controllers import ControllerAPI
+from fastcs.transports.epics.pva.util import validate_pva_id
+
+
+@pytest.mark.parametrize("id", ["DEVICE", "my-id", "name_1", "ABC-123_xyz"])
+def test_validate_pva_id_accepts_valid(id):
```

Ruff (0.15.13)

[error] Function argument `id` is shadowing a Python builtin

(A002)